install_config/aggregate_logging.adoc

```diff
@@ -12,628 +12,475 @@ toc::[]
 +
-====
 ----
-DAEMON_ARGS=
-TD_AGENT_ARGS="/usr/sbin/td-agent --log /var/log/td-agent/td-agent.log --use-v1-config"
+$ oc secrets new logging-deployer \
+   kibana.crt=/path/to/cert kibana.key=/path/to/key
 ----
-====
 
-. Add the following line to the *_/etc/td-agent/td-agent.conf_* file:
+. Create the deployer link:../admin_guide/service_accounts.html[service account]:
 +
 ====
 ----
-  @include config.d/*.conf
+$ oc create -f - <<API
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: logging-deployer
+secrets:
+- name: logging-deployer
+API
 ----
 ====
-
-. Create the *_/etc/td-agent/config.d/kubernetes.conf_* file with the following
-contents:
++
+Then, ensure the logging-deployer can edit the service. The following example
+uses the `logging` project. Be sure to use the project created above in the
+following example:
 +
 ====
-
-[source,xml]
 ----
-    <source>
-      type tail
-      path /var/lib/docker/containers/*/*-json.log
-      pos_file /var/log/td-agent/tmp/fluentd-docker.pos
-      time_format %Y-%m-%dT%H:%M:%S
-      tag docker.*
-      format json
-      read_from_head true
-    </source>
+$ oc policy add-role-to-user edit \
+          system:serviceaccount:logging:logging-deployer
+----
+====
 
-    <match docker.var.lib.docker.containers.*.*.log>
-      type kubernetes
-      container_id ${tag_parts[5]}
-      tag docker.${name}
-    </match>
+== Deploying the EFK Stack
 
-    <match kubernetes>
-      type copy
-      <store>
-        type forward
-        send_timeout 60s
-        recover_wait 10s
-        heartbeat_interval 1s
-        phi_threshold 16
-        hard_timeout 60s
-        log_level trace
-        require_ack_response true
-        heartbeat_type tcp
-        <server>
-          name logging_name <1>
-          host host_name <2>
-          port 24224
-          weight 60
-        </server>
+Deploy the EFK stack is preformed by link:../dev_guide/templates.html[deploying
+a template]. This ensures all future project configurations will feature the EFK
+stack. The default repository and branch of the template are set to
+origin/master. Change these settings to suit your own instance if desired.
 
-        <secondary>
-          type file
-          path /var/log/td-agent/forward-failed
-        </secondary>
-      </store>
+ifdef::openshift-enterprise[]
+The template is available on install, and can be created with the following command:
 
-      <store>
-        type file
-        path /var/log/td-agent/containers.log
-        time_slice_format %Y%m%d
-        time_slice_wait 10m
-        time_format %Y%m%dT%H%M%S%z
-        compress gzip
-        utc
-      </store>
-    </match>
 ----
-<1> The name for the master that will be used during logging.
-<2> The IP or a DNS resolvable name used to access the master.
-====
+$ oc create -f https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/files/examples/infrastructure-templates/enterprise/logging-deployer.yaml
```

Comment:
For OSE, it's installed by default by the installation utility at `/usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/infrastructure-templates/enterprise/logging-deployer.yaml`. So the command for OSE should be:

```
$ oc create -f /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/infrastructure-templates/enterprise/logging-deployer.yaml
```